webrender/src/renderer.rs

```diff
@@ -2861,7 +2875,7 @@ impl Renderer {
 
             for &mut (_, RenderedDocument { ref mut frame, .. }) in &mut active_documents {
                 self.prepare_gpu_cache(frame);
-                assert!(frame.gpu_cache_frame_id <= self.gpu_cache_frame_id);
+                //assert!(frame.gpu_cache_frame_id <= self.gpu_cache_frame_id);
```

Comment:
I guess this is no longer valid? Should it be removed?

Reply:
Ah, you caught me!
Problem is that this assertion would false-fully trigger when replaying any capture made between the introduction of the assert and #2355. So I temporarily (!) commented it out when I was looking at a problem, and forgot about that when preparing the PR.